versions/1/agent/spec.yml

```diff
@@ -0,0 +1,14 @@
+spec:
+  additionalContents: false
+  contents:
+  - description: Folder containing input definitions
+    type: folder
+    name: input
+    required: true
+    additionalContents: false
+    contents:
+    - description: Config template file for inputs defined in the policy_templates section of the top level manifest
+      type: file
+      pattern: '^.+.yml.hbs$'
+      required: true
+      $ref: "./agent.spec.yml"
```

Comment:
As stated above, the `.yml.hbs` file is not a strict JSON/YAML file if you plan to use placeholders. You can't use (reference) a schema for this.

Reply:
right, thanks